test/error_messages.coffee

```diff
@@ -1929,10 +1929,20 @@ test "`new.target` outside of a function", ->
   '''
 
 test "`new.target` is only allowed meta property", ->
-  assertErrorFormat '''
+  assertErrorFormatAst '''
     -> new.something
   ''', '''
     [stdin]:1:4: error: the only valid meta property for new is new.target
     -> new.something
        ^^^^^^^^^^^^^
   '''
+
+test "`new.target` cannot be assigned", ->
```

Comment:
Before, it wasn't throwing a compilation error when eg assigning to `new.target` (since it was just being treated like a normal object-property access)

So this is a benefit of using a separate grammar construct for `MetaProperty`, since `new.target` shouldn't be able to be used as an "assignable"